### ISSUE TITLE
SQL Server stored procedures

### ISSUE THREAD

**Issue by smarteez17** [ErikEJ/EFCorePowerTools#1407]
I am bring in SP from a db. and looking at the docs https://github.com/ErikEJ/EFCorePowerTools/wiki/Reverse-Engineering#sql-server-stored-procedures I cant find an example 

In other tools I seen that SP are created as method and is access via the dbcontext. Does it do the same here ?

Is there some sample code that show how to call SP geting data and updateing or creating data this will help alot as I not sure how to access it and new to EFCorePowerTools

**Comment by ErikEJ** [comment]
There is sample code at that link !!

Otherwise just create a procedure and try it out by looking at the generated code...

**Comment by smarteez17** [comment]
i watching your video on youtube https://www.youtube.com/watch?v=uph-AGyOd8c and I see it bring a ContextProcedures.cs class but whyen I bring any SP it doesnt create that class

**Comment by ErikEJ** [comment]
Did you select any stored procdures? Works on my PC 😄  - I need a repro!

**Comment by ErikEJ** [comment]
Are you not using SQL Server??
As always, please fill in the requested information when creating an issue!

**Comment by smarteez17** [comment]
yes I did I create 3 new SP a get data , create data and update data 

![image](https://user-images.githubusercontent.com/35128168/170829425-a7d5b8c6-2abb-40af-a9a4-5783adb60424.png)


I only see it only under efpt.config.json file 

![image](https://user-images.githubusercontent.com/35128168/170829465-be31f078-7a01-4f1e-a83c-7eb386ffc0d5.png)


yes I am using SQL Server

**Comment by ErikEJ** [comment]
And no code gets generated? Any errors in the Extensions output window?

**Comment by smarteez17** [comment]
nothing just check no errors

**Comment by ErikEJ** [comment]
Do you have any tables selected? Try to select at least one..

**Comment by smarteez17** [comment]
i selected all the tables and all SP still nothing

**Comment by ErikEJ** [comment]
No clue - Can you share the generated code (privately)?

**Comment by ErikEJ** [comment]
Ping?

**Comment by smarteez17** [comment]
Hi I going to try uninstalling and re installing the tool today and retest and will let you know

**Comment by smarteez17** [comment]
Good news I unstalled and install the version on your site and it working now :) no idea why it wasnt before 

thank you

**Comment by ErikEJ** [comment]
Great!
